patches/core/ungoogled-chromium/disable-webstore-urls.patch

```diff
@@ -185,5 +189,5 @@
 -const char kChromeWebstoreBaseURL[] = "https://chrome.google.com/webstore";
 +const char kChromeWebstoreBaseURL[] = "trk:09:https://chrome.google.com/webstore";
  const char kChromeWebstoreUpdateURL[] =
-     "trk:05:https://clients2.google.com/service/update2/crx";
+     "https://clients2.google.com/service/update2/crx";
```

Comment:
Is the removal of the `trk:` scheme from this url intentional?

Also, the fact that `trk:` is being added to `kChromeWebstoreBaseURL` in this patch instead of in `all-add-trk-prefixes-to-possibly-evil-connections.patch` feels off, but it isn't introduced by this pull request, so it shouldn't be a blocker.

Reply:
It is not. It appears that I missed that.

Reply:
Whoops... The removal of the `trk` scheme was not done in this patch, rather, it was likely removed when you refreshed `all-add-trk-prefixes-to-possibly-evil-connections.patch`. So while this should be corrected, it shouldn't be corrected in this patch.

Reply:
I have fixed this issue in my latest push.